sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -512,6 +513,16 @@ export function isAbsoluteUrl(url: string) {
   return _url.startsWith("sb://") || _url.startsWith("http://") || _url.startsWith("https://");
 }
 
+export function getDispositionType(
```

Comment:
Can you add the `@internal`/`@ignore` doc tags so this doesn't show up in documentation?

Reply:
Good point, thanks.